frontend.java

```diff
@@ -0,0 +1,540 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+
+public class frontend {
+	static int loggedIn = 0;
+	static int type = 0; // if type = 1 its retail, if type=2 its agent
```

Comment:
If type can be one of two type, why didn't you define an enum and use that to reference either "retail" or "agent"?